docs/source/packages/accounts-2fa.md

```diff
@@ -0,0 +1,140 @@
+    Accounts.generate2faActivationQrCode((err, svg) => {
+      if (err) {console.error("...", err);return;}
+      /*
+        the svg can be converted to base64, then be used like: 
+         <img 
+            width="200"
+            src={`data:image/svg+xml;base64,${qrCode}`}
+         />
+      */
+      setQrCode(Buffer.from(svg).toString('base64'));
+    })
+  }}
+>
+  Generate a new code
+</button>
+```
+
+
+At this point, the 2FA won't be activated just yet. Now that the user has access to the codes generated by their authenticator app, you can call the function `Accounts.enableUser2fa`:
+
+{% apibox "Accounts.enableUser2fa" "module":"accounts-base" %}
+
+Called with a code the user will receive from the authenticator app once they read the QR code. This function throws an error on failure. If the code provided is correct, a `type` will be added to the user's `twoFactorAuthentication` object, and now the 2FA will be considered enabled:
+
+```js
+twoFactorAuthetication: {
+  type: "otp",
+  secret: "***",
+}
+```
+
+<h3 id="log-in-with-2fa">Log in with 2FA</h3>
+
+Now that you have a way to allow your users to enable 2FA on their accounts, you can create a login flow based on that.
+
+To verify if a user has or not 2FA enabled you can call the function `Accounts.has2FAEnabled`:
+
+{% apibox "Accounts.has2faEnabled" "module":"accounts-base" %}
+
+With this function, you can verify if the has or not 2FA enabled, and based on this information, you can directly log the user in the 2FA is not enabled, or redirect the user to a place where they can provide a code, in case they do have 2FA enabled.
+
+A way of using it would be:
+
+```js
+<button 
+  onClick={() => {
+    Accounts.has2FAEnabled(username, (err, isEnabled) => {
+      if (err) {
+        console.error("Error verifying if user has 2fa enabled", err);
+        return;
+      }
+
+      if (isEnabled) {
+        // send user to a page or show a component 
+        // where they can provide a 2FA code
+        setShouldAskCode(true);
+        return;
+      }
+      // Normal login when they don't have 2FA enabled.
+      Meteor.loginWithPassword(username, password, error => {
+        if (error) {
+          console.error("Error trying to log in (user without 2fa)", error);
+        }
+      });
+    });
+  }
+}>
+  Login
+</button>
+```
+
+If the user has 2FA enabled, and you try to use the function `Meteor.loginWithPassword`, the login will fail, as the user should provide a token to access the app.
+
+The function you will need to call now to allow the user to log in is `Meteor.loginWithPasswordAnd2faToken`:
+
+{% apibox "Meteor.loginWithPasswordAnd2faToken" %}
+
+Now you will be able to receive a code from the user and this function will verify if the code is valid. If it is, the user will be logged in.
+
+So the call of this function should look something like this:
+
+```js
+<button onClick={() => {
+  Meteor.loginWithPasswordAnd2faToken(username, password, code,error => {
```

Comment:
we have a mix in the codebase and also in the docs calling sometimes `code` and sometimes `token` the 2FA code.

We should use the same name everywhere. It seems that `code` is more common than token in the code and docs.